bot.py

```diff
@@ -56,19 +57,215 @@
+            number_countries = await self.wait_for('message', check=is_valid_number)
+        except asyncio.TimeoutError:
+            return
+    else:
+        number_countries = cpp
+
+    if isinstance(number_countries, str) == False:
+        number_countries = number_countries.content
+    if isinstance(number_players, str) == False:
+        number_players = number_players.content
+
+    if is_number(number_players):
+        pass
+    else:
+        await message.channel.send(message.author.mention + ' Unknown what `' + number_players + '` means in context')
+        return
+    if is_number(number_countries):
+        pass
+    else:
+        await message.channel.send(message.author.mention + ' Unknown what `' + number_countries + '` means in context')
+        return
+
+    unusedCountries = []
+    for g in hoi_dict:
+        unusedCountries.append(g)
+    randomIDArray = []
+    counLen = len(unusedCountries)
+    if notRequireMajor:
+        for x in range(0, int(number_players)):
+            randomIDArray.insert(x, [])
+            for y in range(0, int(number_countries)):
+                exit = False
+                while exit == False:
+                    randomInt = random.randint(1, counLen)
+                    try:
+                        unusedCountries.remove(hoi_dict[randomInt-1])
+                        randomIDArray[x].append(int(hoi_dict[randomInt-1]['ID']))
+                        exit = True
+                    except:
+                        pass
+    else:
+        for x in range(0, int(number_players)):
+            randomIDArray.insert(x, [])
+            hasMajorCountry = False
+            for y in range(0, int(number_countries)):
+                exit = False
+                while exit == False:
+                    randomInt = random.randint(1, counLen)
+                    if hasMajorCountry == False and hoi_dict[randomInt-1]['IsMajor'] == 'True':
+                        try:
+                            unusedCountries.remove(hoi_dict[randomInt-1])
+                            randomIDArray[x].append(int(hoi_dict[randomInt-1]['ID']))
+                            hasMajorCountry = True
+                            exit = True
+                        except:
+                            pass
+                    elif hasMajorCountry:
+                        try:
+                            unusedCountries.remove(hoi_dict[randomInt-1])
+                            randomIDArray[x].append(int(hoi_dict[randomInt-1]['ID']))
+                            exit = True
+                        except:
+                            pass
+
+    for p in range(0, len(randomIDArray)):
+        await displayEmbedCountries(message, p, randomIDArray[p])
+
+
 class MyClientBot(discord.Client):
 
     async def on_ready(self):
         print('Logged in as')
         print(self.user.name)
-        print(self.user.id)
         print('------')
-        print('Loading civs: ')
+        print('Loading civs')
```

Comment:
What are all these print statements?  If they are for debugging, then create a logging function, and send the messages to that.  Then at run time you have the option to either log to stdout, or handle debugging messages in a robust way.